oaknational/Oak-Web-Application
# PUPIL-604 feat(quiz): refreshing quiz not erroring

## Description

Music year: 1991

quiz page erroring on refresh for pupil experience;

The issue is the oak-component OakToolTip component

the nextjs error recomended [this approach](https://nextjs.org/docs/messages/react-hydration-error#solution-2-disabling-ssr-on-specific-components)

using next/dynamic and then importing the component and disabling SSR for that specific component

this worked.

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2366--oak-web-application.netlify.thenational.academy/pupils/l/revise-angles-polygons-bearings-0ac0/lessons/find-missing-angles-around-a-point-and-on-a-straight-line-c9k32c/exit-quiz
2. refresh the page
3. You should see the quiz rendered and no errror

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
